R/add_difference.R

```diff
@@ -85,7 +85,7 @@ add_difference.tbl_summary <- function(x,
                                        ...) {
   set_cli_abort_call()
   # check/process inputs -------------------------------------------------------
-  check_dots_empty()
+  check_dots_empty(call = get_cli_abort_call())
```

Comment:
isnt it set just above?

Reply:
is it because there is the generic?

Reply:
> isnt it set just above?

Yep, this is set above, and the `check_dots_empty()` row is a check that throws an error when dots are not empty and we're telling the function which env to reference in the error.